.eslintrc.yml

```diff
@@ -1,3 +1,5 @@
-extends: 'eslint-config-node-services'
+extends: 'wikimedia/server'
 rules:
-  no-console: 'off'
\ No newline at end of file
+  no-console: 'off'
+  no-underscore-dangle: 'off'
+  camelcase: 'off'
```

Comment:
Let's not change the indentation, I think the overrides we've used in other services were quite reasonable, so let's use the same here.

Reply:
I replaced the tabs with spaces but left the rule turned on.